.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+* @a11r @ctiller @nicolas-noble
+bazel/* @a11r @ctiller @nicolas-noble
+cmake/* @a11r @ctiller @jtattermusch @nicolas-noble
+doc/PROTOCOL-HTTP2.md @a11r @ctiller @ejona86 @nicolas-noble
+doc/interop-test-descriptions.md @a11r @ctiller @ejona86 @nicolas-noble
+etc/* @a11r @ctiller @jboeuf @nicolas-noble
+include/* @a11r @ctiller @dgquintas @markroth @nicolas-noble
+src/core/* @a11r @ctiller @dgquintas @markroth @nicolas-noble
+src/cpp/* @a11r @ctiller @dgquintas @markroth @nicolas-noble
+src/csharp/* @a11r @apolcyn @ctiller @jtattermusch @nicolas-noble
```

Comment:
not important now, but I really want myself (and alex) to show up as a first reviewer option for C# PRs. It would be good to understand if ordering on this line influences the order of github approver suggestions.

Also, with this layout it looks like the top-level-owners will start getting way too many review requests.